tests/e2e/homework/components/basePage.ts

```diff
@@ -0,0 +1,17 @@
+import {Page, Locator} from '@playwright/test';
+
+export class BasePage {
+    readonly page: Page;
+    readonly errorAlert: Locator;
+    readonly pageHeader: Locator;
+    readonly button: Locator;
+    readonly successAlert: Locator;
+
+    constructor (page: Page) {
+        this.page = page;
+        this.errorAlert = page.locator('#flash_alert');
+        this.pageHeader = page.locator('h2');
+        this.button = page.locator('input[type="submit"]');
```

Comment:
In terms of generalization - it's ok.
But in terms of test - we will not notice, if label of the button is changed.

And the name is not clear, especially when used in next pages -> button, what button, what that button does? it should be clear from it's name.
How could you solve this issue?